CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 🚨 *Breaking Changes*
 
+* The `orquestra-sdk` version used to submit a workflow is automatically added as a dependency for task execution environments. Consequently, specifying the SDK as a dependency in the `sdk.task()` decorator is no longer supported, and such declarations will be ignored.
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏This was never supported, I'd rephrase this.

Reply:
How about:
```suggestion
* The `orquestra-sdk` version used to submit a workflow is automatically added as a dependency for task execution environments. Specifying the SDK as a dependency in the `sdk.task()` decorator will be ignored.
```